src/tests/ftest/util/server_utils_params.py

```diff
@@ -466,7 +466,6 @@ def __init__(self, base_namespace, index, provider=None, max_storage_tiers=MAX_S
         # Use environment variables to get default parameters
         default_interface = os.environ.get("DAOS_TEST_FABRIC_IFACE", "eth0")
         default_port = int(os.environ.get("D_PORT", 31416))
-        default_share_addr = int(os.environ.get("CRT_CTX_SHARE_ADDR", 0))
```

Comment:
Below:
> undefined-variable, Undefined variable 'default_share_addr'

Maybe we need to remove `crt_ctx_share_addr` entirely?

Reply:
Yeah i will probably do further removal in subsequent patch. I wanted to leave around traces in case we need to add it back in future, but given that we likely won't expose this like that anymore it's probably best to remove most mentions of 'share_addr'.

Reply:
IMO we could just remove it all in this one patch. And if we need it again we can reference this patch and revert that part? The trouble with leaving unused code is that is will eventually break when we do try to use it again

Reply:
Removed more mentions this time around